lint-staged.config.mjs

```diff
@@ -0,0 +1,3 @@
+export default {
+  "*": (filenames) => [`eslint ${filenames.join(" ")} --fix`, "git add"],
+};
```

Comment:
`Git add` command isn't needed and I also think this can be simplified further but that's fine for now:

```suggestion
export default {
  "*": (filenames) => `eslint ${filenames.join(" ")} --fix`,
};
```